lib/Wikia/src/Persistence/User/Attributes/AttributePersistenceSwagger.php

```diff
@@ -107,7 +107,11 @@ public function deleteAttribute( $userId, Attribute $attribute ) {
 	 * @return UsersAttributesApi
 	 */
 	private function getApi( $userId ) {
-		return $this->apiProvider->getAuthenticatedApi( self::SERVICE_NAME, $userId, UsersAttributesApi::class );
+		/** @var UsersAttributesApi $apiClient */
+		$apiClient = $this->apiProvider->getAuthenticatedApi( self::SERVICE_NAME, $userId, UsersAttributesApi::class );
+		$apiClient->getApiClient()->getConfig()->addDefaultHeader("X-From-MW", "true");
```

Comment:
Adding a header so the User Attribute service can identify if a request came from MW or not.